privacyidea/app.py

```diff
@@ -218,18 +234,62 @@ def create_app(config_name="development",
         DEFAULT_LOGGING_CONFIG["loggers"]["privacyidea"]["level"] = level
         logging.config.dictConfig(DEFAULT_LOGGING_CONFIG)
 
-    babel = Babel(app)
-
-    @babel.localeselector
-    def get_locale():
-        return get_accepted_language(request)
-
     queue.register_app(app)
 
-    if init_hsm:
+    if initialize_hsm:
         with app.app_context():
             init_hsm()
 
+    # check that we have a correct node_name -> UUID relation
+    with app.app_context():
+        # first check if we have a UUID in the config file which takes precedence
+        pi_uuid = app.config.get("PI_UUID")
```

Comment:
If an admin enters a UUID in the config file, we read this without logic check. When reading from the PI_UUID_FILE you do a `uuid.UUID()`.

Would it make sense to also add this here:

    pi_uuid = uuid.UUID(app.config.get("PI_UUID"))

otherwise an admin could enter an non-uuid string in pi.cfg and it would be written to the database.

Reply:
correct, will do

Reply:
Done in 4dc04918e